OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -586,6 +586,7 @@
     <Compile Include="Traits\World\WeatherOverlay.cs" />
     <Compile Include="Traits\World\ActorSpawnManager.cs" />
     <Compile Include="Traits\ActorSpawner.cs" />
+    <Compile Include="UpdateRules\Rules\20180923\ChangeTakeOffSoundAndLandingSound.cs" />
```

Comment:
Please revert all other changes to this file and add this change to the last commit. (Otherwise AppVeyor errors as this file is not present in the second commit.)